lua/codeium/views/auth-menu.lua

```diff
@@ -69,7 +61,6 @@ local function open_buffer(url, callback)
 	}
 
 	win_id = vim.api.nvim_open_win(bufnr, true, win_opts)
-	vim.api.nvim_win_set_option(win_id, "cursorline", false)
```

Comment:
What's this deletion for?

Reply:
`win_opts` contains an option `style=minimal` which disables `cursorline` in the window, so no need to do it again